index.html

```diff
@@ -246,7 +246,7 @@ <h1>Let's chat</h1>
                     <input type="email" id="email" name="email" class="email1" placeholder="Email" required>
                     <textarea name="message" id="message" class="text" cols="30" rows="10" placeholder="Enter text here." maxlength="500" required></textarea>
                     <span class="error" aria-live="polite"></span>
-                    <button type="submit" class="btn" onclick="">Submit</button>
+                    <button type="submit" class="btn" onclick="content();">Submit</button>
                     
                 </form>
             </div>    
```

Comment:
[OPTIONAL] I noticed that your form submits when your name input is empty. It should also be marked as required but this is a requirement from previous activity. Please kindly fix it.